torch/distributed/rpc/api.py

```diff
@@ -58,6 +59,29 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+# States used by `def _wait_all_workers()`.
+_ALL_WORKER_NAMES = None
+_INTENDED_WORKER_NAMES = set()
```

Comment:
nit: should call this `SHUTDOWN_INTENT_WORKER_NAMES`

Reply:
That depends? Will `_wai_all_workers()` always used by shutdown?

Reply:
For now its just used in shutdown, we can revisit later as needed.

Reply:
@pritamdamania 
no problem. Let me rename and explain these state vars.